lib/storage-handler.js

```diff
@@ -253,6 +253,7 @@ exports.download = function(provider, req, res, container, file, cb) {
     reader.pipe(res);
 
     reader.on('error', function onReaderError(err) {
+      res.set('content-type', 'application/json');
```

Comment:
>perhaps i should also pass res into processError() then set it there? 
@davidcheung I'm not clear what you mean by passing `res` into `processError()`... could you please elaborate? AFAICT, `processError(err, fileName)` accepts `err` and `fileName`.